WORKSPACE

```diff
@@ -454,3 +454,255 @@ git_repository(
     remote = "https://github.com/istio/test-infra.git",
 )
 
+##
+## Zipkin opentracing implementation. Apparently bazel refuses to synthesize BUILD files
```

Comment:
i think we can remove some of the emotional commentary here ;). Maybe something like "use of underscore in repo requires separate build issue".

It is probably also worth filing an issue with the zipkin repo to ask why they need an underscore and if they would consider not using one. You might even offer to provide them with the bazel files for their repo (and save us embedding this whole mess to begin with).

Also, do we need the "_thrift" stuff?  Can we just build around it?